addons/trenches/functions/fnc_automaticFilePath.sqf

```diff
@@ -0,0 +1,41 @@
+#include "script_component.hpp"
+/*
+ * Author: Salbei
+ * Automatic creation of a path to surface texture.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [TrenchObj, ACE_player] call ace_trenches_fnc_automaticFilePath
+ *
+ * Public: No
+ */
+
+if ((isText (configFile >> "CfgWorlds" >> worldName >> "surfaceTextureBasePath")) || ("surfaceTexture" in (uiNamespace getVariable ["Intercept_cba_capabilities",[]]))) exitWith {};
```

Comment:
This file could use some commenting on what's going on.

Reply:
It tries to generate the path to the surface texture if `surfaceTextureBasePath` doesn't exist.
I don't know what the intercept check is for, I would imagine that this would just not be called if the intercept surfaceTexture is used.